tests/alwaysOnTop.test.js

```diff
@@ -56,24 +87,100 @@ describe('Tests for Always on top', () => {
         });
     });
 
+    it('should bring the app to front in windows', (done) => {
+        if (!isMac) {
+            app.browserWindow.focus();
+            app.browserWindow.restore();
+            app.browserWindow.setAlwaysOnTop(true).then(() => {
+                app.browserWindow.isAlwaysOnTop().then((isOnTop) => {
+                    app.browserWindow.getBounds().then((bounds) => {
+                        robot.setMouseDelay(200);
+                        app.browserWindow.restore().then(() => {
+                            let x = bounds.x + 95;
+                            let y = bounds.y + 35;
+
+                            robot.moveMouseSmooth(x, y);
+                            robot.mouseClick();
+                            robot.setKeyboardDelay(200);
+                            for (let i = 0; i < 4
+                                ; i++) {
+                                robot.keyTap('down');
```

Comment:
We need to figure out a better way to handle this. What if the menu option changes in the future? For now I'll approve this, but if there is a better way, we should look for it.